clients/admin-ui/next.config.js

```diff
@@ -8,16 +8,6 @@ const withBundleAnalyzer = require("@next/bundle-analyzer")({
 const nextConfig = {
     reactStrictMode: true,
     webpack(config) {
-        Object.assign(config.resolve.alias, {
```

Comment:
out of curiosity, what was the motivation for removing this? I don't actually know what it does, so just wondering if you figured it out

Reply:
Basically, we can't mess with how turbo is resolving the `node_modules`, as it has a broader view from the `clients` directory. Without removing these lines, we'll run into the following err upon build (this is an example err from PC, but the same will apply to admin-ui): 

```
privacy-center:build: Failed to compile.
privacy-center:build: 
privacy-center:build: ../node_modules/@chakra-ui/accordion/dist/chakra-ui-accordion.esm.js
privacy-center:build: Module not found: Can't resolve 'react' in '/Users/neville/git/fides/clients/node_modules/@chakra-ui/accordion/dist'
```